ts/Extensions/Annotations/Popup.ts

```diff
@@ -194,15 +194,17 @@ const indexFilter = /\d/g,
  * @private
  */
 enum DropdownProperties {
-    'params.algorithm'
+    'params.algorithm',
+    'params.average'
 }
 
 /**
  * List of available algorithms for the specific indicator.
  * @private
  */
 const dropdownParameters: Highcharts.DropdownParameters = {
-    'algorithm-pivotpoints': ['standard', 'fibonacci', 'camarilla']
+    'algorithm-pivotpoints': ['standard', 'fibonacci', 'camarilla'],
+    'average-disparityindex': ['sma', 'ema']
```

Comment:
The available base indicators are:
- sma
- ema
- dema
- tema
- wma

Shouldn't we add all of them?

Reply:
I will also change `|` into `,` in documentation because it misled me.
![Screenshot 2021-09-16 at 09 54 08](https://user-images.githubusercontent.com/57004120/133572932-5425821a-c25f-4874-b4bd-fd1d09511895.png)

Reply:
What about this?
If a value is different than `'ema'`, `'dema'`, `'tema'` or `'wma'`, then `'sma'` is used.